packages/web-components/fast-components/src/slider/slider.ts

```diff
@@ -254,9 +254,13 @@ export class Slider extends FormAssociated<HTMLInputElement>
         if (this.readOnly || this.disabled || e.defaultPrevented) {
             return;
         }
+
         // update the value based on current position
         const eventValue: number =
-            this.orientation === Orientation.horizontal ? e.pageX : e.pageY;
+            this.orientation === Orientation.horizontal
+                ? e.pageX - this.getBoundingClientRect().left
+                : e.pageY;
+
```

Comment:
Tiny wrench here is #3122 - the entire pathing here is about to change as these are moving into a separate package. Can we hold for #3122?

Reply:
This is currently blocking my progress as it makes it difficult to change the slider when it's positioned on the page. I'm fine with re-doing it for #3122 if the timeline brings it in sooner rather than later - I could also make this PR against that feature branch maybe?

Reply:
If we can land a solution for this with @nicholasrice sometime today I'm good having this go in. The changes in #3122 are significant and I've got a good handle on everything there it's likely safer for you to make the change against master prior if we can. If we don't have a solution and 3122 goes in first we can work together to make sure the rebase is all good.